pkgs/by-name/ri/rimgo/package.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  fetchFromGitea,
+  buildGoModule,
+  tailwindcss,
+}:
+buildGoModule rec {
```

Comment:
```suggestion
}:

buildGoModule rec {
```

Reply:
why useless spacing?

Reply:
Readability. It helps separate the blocks of information.

Reply:
i don't see the necessity for that in a 30 line long file
indentation and syntax highlighting do the job